lib/blocked_edits_monitor.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class BlockedEditsMonitor
```

Comment:
Maybe instead, 'BlockedEditsReporter', since unless like the other similar classes, this one does not do any monitoring; it's job is only to create and email new alerts.